tests/integration/connections/test_snowflake_connector.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from databricks.labs.remorph.connections.database_manager import SnowflakeConnector
+from .helpers import get_db_manager
+
+
+@pytest.fixture()
+def db_manager(mock_credentials):
+    return get_db_manager("remorph", "snowflake")
+
+
+def test_snowflake_connector_connection(db_manager):
+    assert isinstance(db_manager.connector, SnowflakeConnector)
+
+
+def test_snowflake_connector_execute_query(db_manager):
```

Comment:
@sundarshankar89 I added a few integration tests against the connector, modeled after the MSSQL connector tests.

Reply:
You might have to update the `get_db_manager` debug.json gets set a env variables inside the integration test suite. inorder to create the right snowflake or postgres connection manager.